### DISCUSSION TITLE
最新版猫抓一直显示数据载入中

### DISCUSSION THREAD

**Discussion by RimuruOvO** [xifangczy/cat-catch#332] [Q&A]
edge安装的最新版猫抓一直显示数据载入中无法使用，即使重装也不能奏效，辛苦修复一下，这个插件对我真的很重要！！麻烦作者大大了！！！

**Comment by xifangczy** [comment]
在弹出页面 右键 检查 控制台 然后看看有没有错误输出。

**Comment by RimuruOvO** [comment]
您好，猫抓控制台的截图在附件，麻烦了！










在 2024-04-03 23:29:39，"O2bmm" ***@***.***> 写道：

在弹出页面 右键 检查 控制台 然后看看有没有错误输出。

**Comment by RimuruOvO** [comment]
![QQ截图20240404120812](https://github.com/xifangczy/cat-catch/assets/162813494/7003d1f2-ebe7-4878-a7d6-abcba812acdf)

**Comment by xifangczy** [comment]
https://stackoverflow.com/questions/77112781/chrome-extension-error-manifest-000001-unable-to-create-sequential-file
这不是扩展问题。可能某些原因导致删除了储存文件。。你可以按照这个帖子得方法尝试一下重装扩展。
